### ISSUE TITLE
Name your service. Create a service. aria-required-missing

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#66]
The service name field is required but missing the aria-required attribute. 

![image](https://user-images.githubusercontent.com/5230720/136033700-01dcf637-84f9-4737-9536-9e4913d55a34.png)

## Potential fix
The "aria-required" attribute can be set without changing the visual appearance of the form.  

## Resources
https://axe.deque.com/rule-help/en/aria-required-missing"
